Update service programming model documentation for the IGNORED messag… (#3603)
kyma-project/kyma: docs/event-bus/docs/03-03-service-programming-model.md

## Successful Delivery

A message delivered to a subscriber is considered successfully consumed if the service's HTTP response status code is
`2xx`, for example:
```json
{
    "event-id": "22ae22a4-f5b7-4fa1-ada9-558a10a96f3d",
    "status": "published",
    "reason": "Message successfully published to the channel"
}
```

> If the status code is not `2xx` (< 200 or >= 300), it means that a message consumption is not successful and

Reviewer: ```suggestion
If the status code is not `2xx` (< `200` or >= `300`), it means that the message was not consumed successfully and
```

Author: OK

> that the message delivery is re-tried. This implies **At-least-once** delivery guarantee.

Reviewer: ```suggestion
that the Event Bus will try to deliver it again. This implies **At-least-once** delivery guarantee.
```

Author: OK

> Note: If there were no subscriptions or consumers to this `event-type`, the message will be ignored and the response

Reviewer: >**NOTE**:
```suggestion
>**NOTE**: If there were no subscriptions or consumers to this `event-type`, the message is ignored and the response
```

Author: OK

> would be like:

Reviewer: ```suggestion
will look like this:
```

Author: OK
